CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
+## Next
+- Add support for dividing lengthes by speed
+
 ## 1.7.0 - 2021 Jul 14
 - Add operator words `plus`, `minus` and `times`
 - Add operator phrases `multiplied by` and `divided by`
 - Add operator symbol `÷`
+- Add support for dividing length units by speed units (like 10 km / 100 kph)
```

Comment:
Looks like this line was added accidentally. Other than that, everything's good

Reply:
Fixed